coalib/output/ConsoleInteraction.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import platform
 import os
+import copy
```

Comment:
Wait. Can you put this alphabetically where "c" should be? I know platform and os are  not, but i think we should have them sorted